src/views/Dashboard/components/DashboardCards/AuthorizedStudies/index.tsx

```diff
@@ -23,16 +24,34 @@ import styles from './index.module.scss';
 
 const AuthorizedStudies = ({ id, className = '' }: DashboardCardProps) => {
   const dispatch = useDispatch();
-  const { loadingStudiesForFences, fenceStudiesAcls, isConnected, hasErrors, connectionLoading } =
-    useFenceStudies();
+  const {
```

Comment:
I think it needs to be refactored, but this hack can accommodate for multiple fences at once. But the component was not really intended for that I think.